### PR TITLE
Prod 9232: fuel ui nics for fuel 10

### PR DESCRIPTION
**Pull request by rkhozinov**

### CODE REVIEW

File: servers/partner-ci/10.0/ui.yaml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+          wipe-workspace: true
+          skip-tag: true
+    wrappers:
+      - test-wrapper
+    properties:
+      - test-property
+    builders:
+      - get-props
+      - get-ui-refs
+#     - get-updates
```
Comment (naumvd95) [inline servers/partner-ci/10.0/ui.yaml]: why u disable this?
update master is true as seems above

File: servers/partner-ci/common.yaml [hunk 2]
```diff
@@ -967,6 +1002,16 @@
             - "*.log"
           dirmatch: true
 
+- property:
```
Comment (naumvd95) [inline servers/partner-ci/common.yaml:1051]: why dont u use common property

File: servers/partner-ci/10.0/ui.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/10.0/ui.yaml]: variables from  the **get-updates** macros aren't needed for fuel10

File: servers/partner-ci/common.yaml (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/common.yaml:1051]: jobs blocking isn't used for ui

**Review by naumvd95 — APPROVED** [review]

**Pull request merged**